### PR TITLE
Fixed frontend-takehome typo, and made it generic for full timers

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**

### CODE REVIEW

File: interviews/README.md [hunk 1]
```diff
@@ -11,8 +11,8 @@ We don't set a minimum score for any opening, and although we take these criteri
 
 Here are the specific grading rubrics we use to evaluate each of our takehome assignments:
 
-- [Intern Backend Takehome](./frontend-takehome.md)
-- [Intern Frontend Takehome](./backend-takehome.md)
+- [Backend Takehome](./frontend-takehome.md)
```
Comment (borisonr) [inline interviews/README.md]: lol

Comment (JoshuaKGoldberg) [reply interviews/README.md]: oh wait dangit

Comment (borisonr) [reply interviews/README.md]: these should be swapped

File: interviews/frontend-takehome.md [hunk 2]
```diff
@@ -43,8 +43,8 @@ See [Frontend Specifics](#frontend-specifics) below.
 
 Non-core features encouraged for completion after the core features.
 
-- 💖 +2: One piece of delighter content completed fully and Solution Completion is 💖4
-- 👍 +1: One delighter completed with bugs, or completed well but Solution Completion is 👍3
+- 💖 +2: Requested delighter content completed fully and Solution Completion is 💖4
```
Comment (borisonr) [inline interviews/frontend-takehome.md]: 🎉

**Pull request merged**